strings/strings.json

```diff
@@ -7384,5 +7424,9 @@
   "DisplayResultsAs": {
     "en": "Display results as {0}",
     "fr": "Afficher les résultats comme {0}"
+  },
+  "FacetTitle": {
```

Comment:
Use the already translated keyword : `Title`

Reply:
@olamothe, for this one, I prefer to keep the FacetTitle which allows including the title in the string as the `{0}` position. Is that fine? or do you still prefer to use `Title`?